src/main/resources/MessagingDescriptor.json

```
@@ -5,8 +5,14 @@
       "description": "Patron fee/fine balance was changed",
       "eventTTL": 1,
       "signed": false
+    },
+    {
+      "eventType": "LOAN_RELATED_FEE_FINE_CLOSED",
+      "description": "An account with a loan associated is closed",
```

Given we are avoiding the term `account` in the event name, I think we could also avoid it in the description.

I think `is closed` is a statement about the current state, whereas `has been` might be more appropriate for an event, given they represent facts about what has happened in the past.

Does that make sense? What do you think?

```suggestion
      "description": "A fee or fine with a loan associated has been closed",
```

---

Agree, updated.